hack/dep-update.sh

```diff
@@ -29,7 +29,7 @@ done
     echo $_sync_only
     cd staging/src/kubevirt.io/client-go
     if [ "${_sync_only}" == "false" ]; then go get $@ ./...; fi
-    go mod tidy
+    go mod tidy -go=1.16 && go mod tidy -go=1.17
```

Comment:
Please create a separate commit for that purpose, describing the motivation for that change

Reply:
used while updating the deps, no longer needed, removing

Reply:
@xpivarc That was resolved and therefore misleading. @machadovilaca I expect from resolved things to be indeed resolved